jetstream/external_config.py

```diff
@@ -37,32 +36,29 @@ class ExternalConfigCollection:
 
     configs: List[ExternalConfig] = attr.Factory(list)
 
-    JETSTREAM_CONFIG_REPO = "mozilla/jetstream-config"
+    JETSTREAM_CONFIG_URL = "https://github.com/mozilla/jetstream-config"
 
     @classmethod
     def from_github_repo(cls) -> "ExternalConfigCollection":
         """Pull in external config files."""
+        # download files to tmp directory
+        with TemporaryDirectory() as tmp_dir:
+            repo = Repo.clone_from(cls.JETSTREAM_CONFIG_URL, tmp_dir)
 
-        g = Github()
-        repo = g.get_repo(cls.JETSTREAM_CONFIG_REPO)
-        files = repo.get_contents("")
+            external_configs = []
 
-        if isinstance(files, ContentFile):
-            files = [files]
+            for config_file in tmp_dir.glob("**/*.toml"):
+                last_modified = list(repo.iter_commits("main", paths=config_file))[0].committed_date
 
-        configs = []
-
-        for file in files:
-            if file.name.endswith(".toml"):
-                commits = repo.get_commits(path=file.path)
-
-                if commits.totalCount:
-                    slug = os.path.splitext(file.name)[0]
-                    spec = AnalysisSpec.from_dict(toml.loads(file.decoded_content.decode("utf-8")))
-                    last_modified = UTC.localize(commits[0].commit.committer.date)
-                    configs.append(ExternalConfig(slug, spec, last_modified))
+                external_configs.append(
+                    ExternalConfig(
+                        config_file.stem,
+                        AnalysisSpec.from_dict(toml.load(config_file)),
+                        UTC.localize(dt.datetime.utcfromtimestamp(last_modified)),
+                    )
+                )
 
-        return cls(configs)
+            return cls(external_configs)
```

Comment:
I guess this is just aesthetic

```suggestion
        return cls(external_configs)
```